tests/common/ApiDefaultResponses.h

```diff
@@ -29,17 +29,16 @@ namespace mockserver {
 
 class ApiDefaultResponses {
  public:
-  static olp::client::Apis GenerateResourceApisResponse(
-      std::string catalog) {
+  static olp::client::Apis GenerateResourceApisResponse(std::string catalog) {
```

Comment:
Why did you change the versions? `notification`, `publish` and `stream` are v2, see https://developer.here.com/documentation/data-api/api-reference.html. You need to provide exactly the same numbers we expect in the APIs, else our requests will not work.